mappings/net/minecraft/world/EntityDetector.mapping

```diff
@@ -0,0 +1,27 @@
+CLASS net/minecraft/unmapped/C_kkiexczn net/minecraft/world/EntityDetector
+	FIELD f_etrsolop NON_CREATIVE_NON_SPECTATOR_PLAYERS Lnet/minecraft/unmapped/C_kkiexczn;
+	FIELD f_hkridops SHEEP Lnet/minecraft/unmapped/C_kkiexczn;
+	FIELD f_yumifdey NON_SPECTATING_PLAYERS Lnet/minecraft/unmapped/C_kkiexczn;
+	METHOD detect detect (Lnet/minecraft/unmapped/C_bdwnwhiu;Lnet/minecraft/unmapped/C_kkiexczn$C_ztluwxut;Lnet/minecraft/unmapped/C_hynzadkk;D)Ljava/util/List;
+		ARG 2 selector
+		ARG 4 range
+	METHOD m_bappuynt (Lnet/minecraft/unmapped/C_bdwnwhiu;Lnet/minecraft/unmapped/C_kkiexczn$C_ztluwxut;Lnet/minecraft/unmapped/C_hynzadkk;D)Ljava/util/List;
+		ARG 1 selector
+		ARG 3 range
+	METHOD m_geuazetv (Lnet/minecraft/unmapped/C_bdwnwhiu;Lnet/minecraft/unmapped/C_kkiexczn$C_ztluwxut;Lnet/minecraft/unmapped/C_hynzadkk;D)Ljava/util/List;
+		ARG 1 selector
+		ARG 3 range
+	METHOD m_rfaxpdqf (Lnet/minecraft/unmapped/C_bdwnwhiu;Lnet/minecraft/unmapped/C_kkiexczn$C_ztluwxut;Lnet/minecraft/unmapped/C_hynzadkk;D)Ljava/util/List;
+		ARG 1 selector
+		ARG 3 range
+	CLASS C_ztluwxut EntitySelector
+		FIELD f_upomikid WORLD_ENTITIES_SELECTOR Lnet/minecraft/unmapped/C_kkiexczn$C_ztluwxut;
```

Comment:
why entities plural?

Reply:
As in it selects from the world entities, though the name isn't really clear...

Reply:
I think it could still just be the singular, it would communicate the same and match the classname

Reply:
alright, fixed